### ISSUE TITLE
pagespeed_no_defer generates an error in html5 validation

### ISSUE THREAD

**Issue by ConfusedVorlon** [pagespeed/ngx_pagespeed#991]
site: hobbyistsoftware.com

validation error from 

https://validator.w3.org/nu/?doc=http%3A%2F%2Fhobbyistsoftware.com%2F

Error: Attribute pagespeed_no_defer not allowed on element script at this point.

**Comment by jeffkaufman** [comment]
Yes, we should allow `data-pagespeed_no_defer` as a synonym for `pagespeed_no_defer`.

**Comment by jeffkaufman** [comment]
@jmaessen

**Comment by jmaessen** [comment]
Should be fixed in:
9fb167b..ab76063  master -> master

Jeff, let me know if I should be attaching the commit to the bug in some more formal fashion.

**Comment by crowell** [comment]
If you put "closes #(bug number)" somewhere in the commit message it
automatically will close and reference the bug.

**Comment by jmaessen** [comment]
OK, how do I do that post facto?

**Comment by crowell** [comment]
commenting on the bug "closed by (hash)" links them in gihubs ui

**Comment by jmaessen** [comment]
Ah, but of course that was a mainline commit.
